components/layout_2020/flexbox/layout.rs

```diff
@@ -220,7 +220,7 @@ impl FlexContainer {
         let mut item_infos = vec![];
 
         let container_is_horizontal = self.style.writing_mode.is_horizontal();
-        let flex_direction = used_flex_direction(&self.style);
+        let flex_direction = used_flex_direction(&*self.style);
```

Comment:
I think this change is not needed because the code used to compile before (due to deref coersion).

Reply:
Hrm. I think this was a weird merge issue. I've resolved it now.